packages/react/migrations.json

```diff
@@ -193,6 +193,15 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "19.0.5": {
+      "version": "19.0.5-beta.0",
```

Comment:
Can you update the version here? `19.1.1` if you think you can get this in today?